### PR TITLE
add homepage for bebop.sh

### PR DESCRIPTION
**Pull request by kushsolitary**

### CODE REVIEW

File: homepage/logo-animation.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+let preElem = document.querySelector('.logo-code')
+let originalCode = preElem.innerText
+
+console.log(originalCode)
+
```
Comment (AndrewMD5) [inline homepage/logo-animation.js:5]: Probably no need to write this to the console.

File: homepage/index.html [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+            :XMMMMMMKdc;;:cok0d. ;KMMMMMMMMMMMMMMMMMMWx.                                       
+            .oXMMMMMMMMWWMMMWKc  '0WMMMMMMMMMMMMMMMMWk'                                        
+              'oOXWMMMMMMWXOl.    .lOXWMMMMMMMMMMWKx:.                                         
+                .;ok0KK0ko;.        .;d0XWMMWWNKxc'                                           
+</pre>
+            </div>
+        </div>
+
+        <div class="container">
+            <h1>An efficient, schema-based binary serialization format</h1>
```
Comment (AndrewMD5) [inline homepage/index.html:57]: Can you change this to An extremely simple, fast, efficient, cross-platform serialization format

File: homepage/logo-animation.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        else {
+            s[i] = newCharacters[Math.floor(Math.random() * newCharacters.length)]
+            newCode += s[i]
+        }
+    }
+
+    preElem.innerText = newCode
+}
+
+setInterval(animateCode, 200)
```
Comment (AndrewMD5) [inline homepage/logo-animation.js:29]: Add semi-colons just because we use them in all our other Javascript files.

**Review by AndrewMD5 — CHANGES_REQUESTED** [review]

**Comment by kushsolitary** [comment]
@AndrewMD5 changes done

**Review by AndrewMD5 — APPROVED** [review]

**Pull request merged**